makejdk.sh

```diff
@@ -260,12 +260,9 @@ buildAndTestOpenJDKViaDocker()
 
 testOpenJDKInNativeEnvironmentIfExpected()
 {
-  if [[ ! -z $JTREG ]]; then
-    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+  if [[ "${JTREG}" == "true" ]];
+  then
```

Comment:
@neomatrix369 I agree with @gdams here, I think we can go with:

`if [[ ${JTREG} == "true" ]];`

We can fix the other scripts later